### PR TITLE
Create release-notes.md file

### PR DESCRIPTION
**Pull request by simrunm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,7 +254,7 @@ jobs:
       - run: npm test
 
       - run: npm run release
-        with:
+        env:
```
Comment (jbcpollak) [inline README.md]: is this change intentional?

File: src/commands/publish.ts [hunk 2]
```diff
@@ -158,6 +159,7 @@ export class Publish extends Command<Argv> {
       await this.createReleaseTag()
       await this.pushToRemote()
```
Comment (jbcpollak) [inline src/commands/publish.ts]: this needs to be commented out

**Review by jbcpollak — COMMENTED** [review]
two comments

File: README.md (hunk 1 above)
Comment (simrunm) [reply README.md]: no, I thought you made that one but I'm switching it back

**Review by jbcpollak — APPROVED** [review]
